src/Core/tests/Benchmarks.Droid/Benchmarks.Droid.csproj

```diff
@@ -28,7 +28,7 @@
     <Using Include="BenchmarkDotNet.Order" />
     <Using Include="BenchmarkDotNet.Running" />
     <PackageReference Include="BenchmarkDotNet" Version="0.13.10" />
-    <PackageReference Include="Xamarin.Android.Glide" Version="4.14.2.1" />
```

Comment:
This version was not in sync with the one we use on all other projects.

Reply:
Can we put the entry here across the repo?
* https://github.com/dotnet/maui/blob/main/eng/AndroidX.targets

Then you could just put `<PackageReference Include="Xamarin.Android.Glide" />` without the `Version` across the repo.

Reply:
Actually I found something better which is already used in the solution `$(_XamarinAndroidGlideVersion)`.